kodilla-jdbc/src/main/java/com/kodilla/jdbc/DbManager.java

```diff
@@ -0,0 +1,33 @@
+package com.kodilla.jdbc;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class DbManager {
+    private Connection conn;
+    private static DbManager dbManagerInstance;
+
+    private DbManager() throws SQLException {
+        Properties connectionProps = new Properties();
+        connectionProps.put("user", "kodilla_user");
+        connectionProps.put("password", "kodilla_password");
+        conn = DriverManager.getConnection(
+                "jdbc:mysql://localhost:3306/kodilla_course?serverTimezone=Europe/Warsaw" +
+                        "&useSSL=false",
+                connectionProps);
+    }
+
+
+    public static DbManager getInstance() throws SQLException {
```

Comment:
testy są często uruchamiane w środowisku wielowątkowym, więc wtedy istnieje ryzyko stworzenia dwóch instancji DbManagera. 
Popraw proszę to na implementację Singletona, która jest thread safe